deployments/rbac-server/values.yaml

```diff
@@ -74,10 +78,12 @@ roleScopesMap:
   - api.batch.jobs.write
   - api.files.read
   - api.files.write
-  - api.vector-stores.read
-  - api.vector-stores.write
+  - api.vector_stores.read
```

Comment:
Do you plan to update https://github.com/llmariner/vector-store-manager/blob/main/server/internal/server/server.go#L97 ?

Reply:
question: does it need to match with the resource path defined in `proto` and `ingress`?

Reply:
@guangrui-cloudnatix , hm, I think we don't have any specific convention. Changing to `vector_stores` looks more consistent with the rest, but we just need to make the corresponding change to `vector-store-manager`.

Reply:
I see. revert the changes for `vector-stores`